### PR TITLE
#162036182 add an external database to store user and order info

### PR DESCRIPTION
**Pull request by BrianSerem**
#### What does this PR do?
Introduce storage of user info in an external postgresql server. 

#### Description of Task to be completed?
-Create a db_config.py file containing queries for creating users and orders tables
-Create a connection to the Database
-Instantiate a cursor object to execute queries.

#### How should this be manually tested?
Clone this repo and cdd into themain directory.
Run db_config.py
#### Any background context you want to provide?
The database contain two tables, orders nd users.

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/162042923

### CODE REVIEW

File: app/api/V2/utils.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    '''validate input'''
+    regex = "^[a-zA-Z0-9_]{1,20}$"
+    return re.match(regex,input)   
+
+
+    
+def valid_destination(destination):
+    '''validate destination name'''
+    regex = "^[a-zA-Z 0-9]{3,}$"
+    return re.match(regex, destination)
```
Comment (cosmas28) [inline app/api/V2/utils.py]: Kindly leave one line space at the end of each file.

File: app/api/V2/utils.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import re
+
+
+def valid_destination_name(destination):
+    ''' validate destination name '''
```
Comment (cosmas28) [inline app/api/V2/utils.py:5]: You should be persistence when using quotes, use either single or double quotes, don't use both.

File: app/tests/V1/test.py [hunk 3]
```diff
@@ -257,6 +257,268 @@ def test_invalid_weight(self):
...
+            "api/v2/parcels",
+            data=json.dumps(data),
+            headers={"content-type": "application/json"}
+        )
+
+        self.assertEqual(res.status_code, 400)
+        self.assertEqual(json.loads(res.data)['message'], {'weight': 'Invalid weight'})
+
+
+if __name__ == '__main__':
```
Comment (cosmas28) [inline app/tests/V1/test.py]: Kindly remove one of this blocks of codes.

File: app/tests/V2/test.py [hunk 4]
```diff
@@ -0,0 +1,262 @@
+"""Contain test cases for the endpoints."""
```
Comment (cosmas28) [inline app/tests/V2/test.py:1]: How comes both `app/tests/v2/test.py` and `app/tests/v1/test.py` have tests for v2 endpoints?

File: app/tests/V1/test.py (hunk 3 above)
Comment (BrianSerem) [reply app/tests/V1/test.py]: I removed one of them, it was a copy/paste error.

File: app/tests/V2/test.py (hunk 4 above)
Comment (BrianSerem) [reply app/tests/V2/test.py:1]: The tests for version 2 should be inside the V2 folder, I have done exactly that, moved all version 2 test cases to a file test.py inside V2. I also added two more tests for testing User registration endpoint.

**Pull request merged**